react_practice/react-apps/waitlist-page-v1/package.json

```diff
@@ -34,5 +35,12 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "prettier": "^2.4.1"
```

Comment:
I went ahead and installed/configured `prettier` so that we can have consistent code styles 🙂